tools/pd-tso-bench/main.go

```diff
@@ -33,9 +33,9 @@ var (
 	pdAddrs     = flag.String("pd", "127.0.0.1:2379", "pd address")
 	concurrency = flag.Int("C", 1000, "concurrency")
```

Comment:
how about lower case

Reply:
This PR will only modify the help message. We can raise another PR to do it if it's necessary.